UtilitsforBioinfApplication/helpertools/dna_rna_tools_module.py

```diff
@@ -0,0 +1,61 @@
+def transcr(seq):  # accepts a string - a sequence of nucleotides of any length
+    newseq = ''
+    for nucl in seq:
+        if nucl == 'T':
+            newseq += 'U'
+        elif nucl == 't':
+            newseq += 'u'
+        else:
+            newseq += nucl
+    return (newseq)  # returns a string - a sequence translated to RNA form
+
+
+def reverse(seq):  # accepts a string - a sequence of nucleotides of any length
+    newseq = ''
+    newseq = seq[::-1]
+    return (newseq)  # returns a string - a reversed sequence
```

Comment:
```suggestion
    return seq[::-1]  # returns a string - a reversed sequence
```

Reply:
There are several places, where after "return" author used brackets - it is not necessary)